core/src/autogluon/core/models/ensemble/bagged_ensemble_model.py

```diff
@@ -356,6 +357,32 @@ def predict_proba(self, X, normalize=None, **kwargs):
 
         return pred_proba
 
+    def predict_folds(self, X, **kwargs):
+        y_pred_probas = self.predict_proba_folds(X, **kwargs)
+        y_preds = []
+        for y_pred_proba in y_pred_probas:
+            y_pred = get_pred_from_proba(y_pred_proba=y_pred_proba, problem_type=self.problem_type)
+            y_preds.append(y_pred)
+        return y_preds
+
+    def predict_proba_folds(self, X, normalize=None, **kwargs):
+        pred_probas = []
+        has_preprocessed = False
+        for i, model in enumerate(self.models):
+            model = self.load_child(model)
+            if not has_preprocessed:
+                X = self.preprocess(X, model=model, **kwargs)
+                has_preprocessed = True
+            pred_proba = model.predict_proba(X=X, preprocess_nonadaptive=False, normalize=normalize)
+
+            if self.params_aux.get("temperature_scalar", None) is not None:
+                pred_proba = self._apply_temperature_scaling(pred_proba)
+            elif self.conformalize is not None:
```

Comment:
Should we allow for _both_ temperature scaling and conformal predictions?

An important note is that most conformal methods will not return calibrated probabilities (like temperature scaling would do), but they will rather return sets of likely predictions. In regression, these sets are usually expressed as intervals around the model prediction. In classification, these are discrete sets of likely variables (e.g. [1, 4, 6], for instance).

A few specific conformal method might be able to return calibrated probabilities, but it is not the standard output of conformal prediction. What output should we aim for?